docs/README.md

```diff
@@ -0,0 +1,116 @@
+# Documentation for Unity Catalog
+
+Unity Catalog uses **MkDocs** to manage and serve its documentation. MkDocs is a simple, static site generator that’s geared towards project documentation and is written in Python. This guide will help you set up the local environment, serve the documentation locally, and deploy it to GitHub Pages.
+
+## Setting Up MkDocs for Local Development
+
+To start testing or modifying the documentation locally, follow these steps:
+
+### 1. Install MkDocs
+
+Make sure you have **MkDocs** installed in your environment. If it’s not installed, you can do so using `pip`.
+
+```bash
+pip install mkdocs
+```
+
+You can verify that it is installed by running:
+
+```bash
+mkdocs --version
+```
+
+### 2. Clone the Repository
+
+Clone the Unity Catalog repository and navigate to the documentation directory.
+
+```bash
+git clone https://github.com/unitycatalog/unitycatalog.git
+cd unitycatalog/docs
+```
+
+### 3. Install the Required Dependencies
+
+The `requirements-docs.txt` file contains all the dependencies necessary for serving the documentation, including **MkDocs** and its plugins. First, create a virtual environment:
+
+```bash
+# Create virtual environment
+python -m venv uc_docs_venv
+
+# Activate virtual environment (Linux/macOS)
+source uc_docs_venv/bin/activate
+
+# Activate virtual environment (Windows)
+uc_docs_venv\Scripts\activate
+```
+
+Next, install the dependencies:
```

Comment:
Suggest updating this to: `Next, install the following dependencies into your activated virtual environment.`.